activerecord/lib/active_record/persistence.rb

```diff
@@ -955,6 +955,10 @@ def belongs_to_touch_method
       :touch
     end
 
+    def has_one_touch_method
```

Comment:
This abstraction is no longer necessary.

Reply:
Fixed this too. 👍